cli/tests/integration/lsp_tests.rs

```diff
@@ -3718,6 +3701,23 @@ export class DuckConfig {
           }]
         }]
       }
+    }, {
+      "title": "Add all missing imports",
+      "kind": "quickfix",
+      "diagnostics": [{
+        "range": {
+          "start": { "line": 0, "character": 50 },
+          "end": { "line": 0, "character": 67 }
+        },
+        "severity": 1,
```

Comment:
Did TypeScript just change the ordering of returned diagnostics or are we sorting these in any way?